File: csa-app-client-spa/src/app/conference-twitter/person.ts
import {Organizer} from "./organizer";
import {Sponsor} from "./sponsor";

export class Person implements Organizer, Sponsor{
  type: string = "person";
  name: string;
  constructor(public firstName: string, public lastName: string) {
    this.name = firstName + " " + lastName;
  }
}

File: csa-conference-collector/src/main/java/net.csa.conference.collector/Person.java
package net.csa.conference.collector;

/**
 * @author philipp.amkreutz
 */
public class Person implements Organizer, Sponsor {

	private String firstName;
	private String lastName;
	private String type = "person";

	public Person() {
	}

	public Person(String firstName, String lastName) {
		this.firstName = firstName;
		this.lastName = lastName;
	}

	public String getName() {
		return "Person: " + firstName + " " + lastName;
	}

	public void setFirstName(String firstName) {
		this.firstName = firstName;
	}

	public void setLastName(String lastName) {
		this.lastName = lastName;
	}

}
